neurons/validator.py

```diff
@@ -981,7 +991,9 @@ def log_step(
         }
 
         # The sub-competition weights
-        sub_competition_weights = torch.softmax(model_weights / constants.temperature, dim=0)
+        sub_competition_weights = torch.softmax(
+            model_weights / constants.temperature, dim=0
+        )
```

Comment:
Github won't let me comment on the relevant line. 

WDYT about adding a colum to the below table that includes the repo in <namespace>/<name> format as well?

Reply:
Added.